_posts/zh/newsletters/2018-08-07-newsletter.md

```diff
@@ -0,0 +1,84 @@
+---
+title: 'Bitcoin Optech Newsletter #7'
+permalink: /zh/newsletters/2018/08/07/
+name: 2018-08-07-newsletter-zh
+slug: 2018-08-07-newsletter-zh
+type: newsletter
+layout: newsletter
+lang: zh
+---
+本周的新闻简报包括常规的仪表板和行动项目，一个关于通过闪电网络进行通用比特币合约讨论的链接，一个最近宣布的用于增强可扩展性的 BLS 签名库的简要描述，以及来自 Bitcoin Core、LND 和 C-Lightning 项目的一些值得注意的提交。
+
+## 行动项目
+
+- Optech 已开始计划其首次欧洲工作坊，预计将于 11 月在巴黎举行。如果任何认为他们将能够参加的会员公司对讨论的主题感兴趣，请[给 Optech 发送电子邮件][optech email]。关于工作坊的更多信息将在几周内发布。
+
+## 仪表板项目
+
```

Comment:
```suggestion
{% assign img1_label = "Transactions signaling opt-in RBF, August 2017 - August 2018" %}
```

Reply:
need to add this label to solve the compiling problem

Reply:
Thanks @hulatown !